### DISCUSSION TITLE
Switch between wallpaper folders (many images)

### DISCUSSION THREAD

**Discussion by mishavoloshchuk** [AutoDarkMode/Windows-Auto-Night-Mode#657] [General]
Hi! Can you add the posibility to set the wallpapers folder for dark/light theme? Windows 10 has such an implementation in personalization settings, but without changing the folder when switching between themes.

**Comment by Spiritreader** [comment]
Hi, we have an enhancement issue open about supporting Windows slideshows (which is a superset of your described feature).
You can find it here:
https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/issues/247
There's also an outline and a to-do list on what needs to be done to get it working.

It's planned, but not likely any time soon in the near future due to lack of time. Unless someone wants to work on it and open a PR, that is.

**Comment by johnsolo1** [comment]
Hi check my post see if that works for you!
